types/array-binarysearch.closest/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for array-binarysearch.closest 0.2
+// Project: https://www.npmjs.com/package/array-binarysearch.closest
+// Definitions by: Heye Vöcking <https://github.com/hvoecking>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function binarySearch<T>(
```

Comment:
This code won't work - the original package is declared with `module.exports = `, so this package should use `export = binarySearch`.